components/vf-article-meta-information/vf-article-meta-information.scss

```diff
@@ -107,9 +106,8 @@
 
 
 .vf-meta__comments {
-
-  .vf-text-body--6 {
+  p {
```

Comment:
does this have to be a vanilla `p` tag, can we do a class or change the html structure?

Reply:
resolved that in last commit - b6fc7cf

Reply:
👍